website/addons/owncloud/serializer.py

```diff
@@ -0,0 +1,68 @@
+from website.addons.base.serializer import StorageAddonSerializer
+from website.addons.owncloud.settings import DEFAULT_HOSTS
+from website.util import web_url_for
+
+from owncloud import Client as OwnCloudClient
+
+class OwnCloudSerializer(StorageAddonSerializer):
+
+    addon_short_name = 'owncloud'
+
+    # Include host information with more informative labels / formatting
+    def serialize_account(self, external_account):
```

Comment:
I'm not sure if this override is necessary --  it breaks some architectural generalizations without providing any more benefit than a docstring would. Consider adding a docstring noting quirks instead.